source/base/function_parser.cc

```diff
@@ -88,9 +79,8 @@ FunctionParser<dim>::FunctionParser(const std::string &expression,
 }
 
 
-// We deliberately delay the definition of the default destructor
-// so that we don't need to include the definition of mu::Parser
-// in the header file.
+
+// See the note in the header for why this is here.
 template <int dim>
 FunctionParser<dim>::~FunctionParser() = default;
```

Comment:
This is where the comment about the destructor should go. Interesting that one can put the `=default` onto a definition -- I hadn't realized this.